Stream-Three/Unit14-Custom_user_and_email_authentication/auth_demo/accounts/models.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import AbstractUser, UserManager
+from django.db import models
+from django.utils import timezone
+
+# Create your models here.
+class AccountUserManager(UserManager):
+    def _create_user(self, username, email, password,
```

Comment:
@NielMc This doesn't seem to be used... 

Aside from that, the [Django documentation](https://docs.djangoproject.com/en/1.8/topics/auth/default/#creating-users) says that the UserManager does have a method called create_user, but this is a private method so it's not overriding the method on UserManager. Unless it's being called implicitly somewhere but I'm not overly familiar with using the UserManager.

@yoniLavi @richardadalton do either of you know any more about this?